META.d/README.md

```diff
@@ -0,0 +1,7 @@
+# Cloud Catalog Information
+
+These documents are meant to feed [Heimdall](https://heimdall.hashicorp.services). A detailed
+breakdown of the schema can be found
+[here](https://github.com/hashicorp/core-sre-heimdall-core/blob/main/docs/SCHEMA.md). There is also
+a guide [on
+Confluence](https://hashicorp.atlassian.net/wiki/spaces/ENSV/pages/2480341525/How-To+Document+your+Heimdall+Asset+with+GitHub).
```

Comment:
Hey I think we should omit this file from this repo since these are all private links.